operations/toc-chair.md

```diff
@@ -0,0 +1,41 @@
+# TOC Chair
+
+This document serves to provide the TOC Chair with details on their responsibilities, role, and expectations that are in addition to those of a TOC member.
+
+## Chair selection and time committment
+
+The TOC Chair is an member of the TOC that is chosen by each seated TOC to lead and resent the TOC in the execution of it's work. The expected time commitment of a TOC member who is serving as Chair is roughly **20-25% of their monthly regular work time**, and may be higher or lower depending on existing community and project challenges, activity of the TOC, and other occurrences that may occupy their time (such as serving in an interim capacity on other groups). The Chair should also anticipate travel to conferences and offsites at least twice a year, with participation and attendence at various meetings during the CNCF flagship conference: KubeCon+CloudNativeCon (KCCN).
+
+## Functions of the Chair
+
+The Chair is expected to perform a variety of functions and can often be thought of as a Team Lead, as such their work relies on the input and support of the TOC to be successful. A list of known functions and their details is below:
+
+* Facilitate Initiative Planning
+* Coordinate TOC activities
+* Ensure Liaisons are assigned and engaged for each TAG
+* Coordinate overarching TAG activities
+* Serve on the Governing Board in the TOC Seat and represent the TOC to the Governing Board
+* Co-present CNCF technical strategy at Governing Board strategy sessions
+* Support onboarding new TOC members as needed
+* Ensure notes and actions are documented and issued for TOC meetings with follow-up
+* Verify appropriate workload distribution among the TOC members
+* Ensure TOC meetings have a facilitator and agenda
+* Foster discussion - ask questions, even obvious ones to move the conversation forward or to closure
+* Protect TOC time - People like to sign the TOC up for a lot of things, you are responsible for ensuring we’re involved in the things we need to be and which ones are not on us
+* Redirect work out of scope of the TOC to the appropriate group (Kubernetes Steering, Governing Board, Code of Conduct Committee, Legal Committee, etc.)
+* Bring project and group conflicts that escalate to the TOC level to closure, either through delegation to TOC members and awareness or through direct resolution while keeping the TOC informed
+* Meet regularly with CNCF Leadership, CNCF Staff Supporting the TOC, and other Foundations' Technical Leadership to coordinate cross-foundation opportunities and awareness
+* Be available to field questions and concerns from community members and redirect as needed
```

Comment:
I feel like in general this shouldn't be a specific task for the TOC chair. Or is it more a matter of monitoring the public TOC Slack and feeling responsible to respond?

Reply:
you are correct, it is not specific to the chair.  I'll ensure its in the TOC area as well. I want to leave it called out here because ultimately someone is responsible for providing a response and the Chair is usually the individual that conveys the consensus of the TOC.  Any TOC member can do it, but in instances where there is a lot of back and forth the chair coordinates and provides the response as the central contact.